modules/core/site.js

```diff
@@ -1802,7 +1802,11 @@ $(function() {
         $('.reset_default_value_select').on("click", reset_default_value_select);
         $('.reset_default_value_input').on("click", reset_default_value_input);
     }
-    
+
+    if (hm_check_dirty_flag()) {
+        $('.add_server, .change_pass form, .add_contact_form, .edit_profile form, .user_settings form, .add_hl_form form, .compose_form').areYouSure();
```

Comment:
Don't we want to do it on an any form on the site? Maybe except the search forms? Otherwise, we risk omitting forms in modules. I doubt this is the complete list of forms we have.

Reply:
I think that's the only issue I find. All the rest looks good!

Reply:
Thanks @kroky. Fixed !